stylesheets/layout/filter.css

```diff
@@ -0,0 +1,91 @@
+.filter-wrapper {
+  flex-grow: 0;
+  flex-shrink: 0;
+  flex-basis: 270px;
+  padding: 2rem 0 0 2rem;
+}
+
+.filter-showcase {
+  position: sticky;
+  padding: 2rem;
+  top: 0;
+  background-color: var(--default-light);
+}
+
+.filter-heading {
+  display: flex;
+  justify-content: space-between;
+}
+
+.filter-heading-left,
+.filter-heading-right {
+  font-size: var(--font-s-md);
+  padding-bottom: 8px;
+}
+
+.filter-heading-left {
+  font-weight: var(--font-w-medium);
+  color: var(--text-color);
+}
+
+.filter-heading-right {
+  font-weight: var(--font-w-medium);
+  color: var(--primary-400);
+  transition: all 0.3s ease-in-out;
+  text-decoration: underline;
+}
+
+.filter-heading-right:hover {
+  color: var(--active-color);
+}
+
+.filter-content {
+  overflow-y: auto;
+  max-height: calc(100vh - 5.5rem);
+}
+
+.filter-subcontent {
+  padding: 1rem 0.5rem;
+}
+
+.filter-subcontent-heading {
+  font-size: var(--font-s-md);
+  font-weight: var(--font-w-medium);
+  color: var(--active-color);
+  padding: 5px 0;
+}
+
+.filter-price {
+  font-size: var(--font-s-lg);
+  font-weight: var(--font-w-medium);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 5px 0;
+  color: var(--white);
+}
+
+.filter-category,
+.filter-rating,
+.filter-sorting,
+.filter-availability {

```

Comment:
just a suggestion not mandatory like use can defined common class name use it everywhere if the usage is common or else if it's specific usage then no need to change good to go

Reply:
Kept it intensionally considering future manipulation.

Reply:
ok np